### DISCUSSION TITLE
Error: "Downtime due to GitHub API rate limiting" on Personal Deployed Instance

### DISCUSSION THREAD

**Discussion by builtbybasit** [anuraghazra/github-readme-stats#3611] [Q&A] [answered]
I am getting Error on my github readme stats deployed instance on Vercel with my own PAT.

![2024-01-23_21-21](https://github.com/anuraghazra/github-readme-stats/assets/45251447/09fd3d7e-a8fc-4675-81cd-a1c7d0e86adf)

```html
<img src="https://github-readme-stats-abdulbasitrana.vercel.app/api?username=basitcodeenv"/>
```

<img src="https://github-readme-stats-abdulbasitrana.vercel.app/api?username=basitcodeenv"/>

Please help!

**Comment by qwerty541** [comment, marked as answer]
Hey, @basitcodeenv! You can check status of API tokens used in your instance by the following link:
https://github-readme-stats-abdulbasitrana.vercel.app/api/status/pat-info
As you see it shows that your current token is expired.

**Reply by builtbybasit** [reply]
I found the problem. I was not redeploying the instance after changing the environment variable.
